52.md

```diff
@@ -0,0 +1,212 @@
+
+This kind of calendar event starts on a date and ends before a different date in the future. Its use is appropriate for all-day or multi-day events where time and time zone hold no significance. e.g., anniversary, public holidays, vacation days.
+
+#### Format
+
+The format uses a parameterized replaceable event kind `31922`.
+
+The `.content` of these events is optional and should be a detailed description of the calendar event.
+
+The list of tags are as follows:
+* `d` (required) universally unique identifier (UUID). Generated by the client creating the calendar event.
+* `summary` (required) name of the calendar event
+* `start` (required) inclusive start date in ISO 8601 format (YYYY-MM-DD). Must be less than `end`, if it exists.
+* `end` (optional) exclusive end date in ISO 8601 format (YYYY-MM-DD). If omitted, the calendar event ends on the same date as `start`.
+* `location` (optional) location of the calendar event. e.g. address, GPS coordinates, meeting room name, link to video call
+* `g` (optional) [geohash](https://en.wikipedia.org/wiki/Geohash) to associate calendar event with a searchable physical location as suggested as an example by [NIP-12](12.md)
+* `p` (optional, repeated) 32-bytes hex pubkey of a participant, optional recommended relay URL, and participant's role in the meeting
+* `t` (optional, repeated) hashtag to categorize calendar event
+* `r` (optional, repeated) references / links to web pages, documents, video calls, recorded videos, etc.
+
+```json
+{
+  "id": <32-bytes lowercase hex-encoded SHA-256 of the the serialized event data>,
+  "pubkey": <32-bytes lowercase hex-encoded public key of the event creator>,
+  "created_at": <Unix timestamp in seconds>,
+  "kind": "31922",
+  "content": "<description of calendar event>",
+  "tags": [
+    ["d", "<UUID>"],
+
+    ["summary", "<summary of calendar event>"],
+
+    // Dates
+    ["start", "<YYYY-MM-DD>"],
+    ["end", "<YYYY-MM-DD>"],
+
+    ["start_tzid", "<IANA Time Zone Database identifier>"],
+    ["end_tzid", "<IANA Time Zone Database identifier>"],
+
+    // Location
+    ["location", "<location>"],
+    ["g", "<geohash>"],
+
+    // Participants
+    ["p", "<32-bytes hex of a pubkey>", "<optional recommended relay URL>", "<role>"],
+    ["p", "<32-bytes hex of a pubkey>", "<optional recommended relay URL>", "<role>"],
+
+    // Hashtags
+    ["t", "<tag>"],
+    ["t", "<tag>"],
+
+    // Reference links
+    ["r", "<url>"],
+    ["r", "<url>"]
+  ]
+}
+```
+
+### Time-Based Calendar Event
+
+This kind of calendar event spans between a start time and end time.
+
+#### Format
+
+The format uses a parameterized replaceable event kind `31923`.
+
+The `.content` of these events is optional and should be a detailed description of the calendar event.
+
+The list of tags are as follows:
+* `d` (required) universally unique identifier (UUID). Generated by the client creating the calendar event.
+* `summary` (required) name of the calendar event
```

Comment:
`name` is more intuitive than `summary` for a calendar event name

Reply:
Agreed. Changed `summary` to `name` in 1058d1bc9cf4259d4386629f3169290dc940052f